tests/test_files/long_expressions.py

```diff
@@ -52,3 +52,8 @@ async def bar():
 
 with weird_container[10:str(datetime.datetime.today().date())[:100], None]:
     pass
+
+
+async def async_for_function():
```

Comment:
I don't understand how this fixture will help us with expression complexity checking. Looks like regular code snippet. Could you explain please?

Reply:
The code was not working properly for `async for` function.  The idea here is to test it.  You can try to add this code on a different branch to see the results.


Not sure I understood what you were saying though...

Reply:
If you see #15, this tool was not able to run the complexity check on a file that contained an `async for` syntax.

Reply:
Ok, understood. I think we should add example with high complexity for `async for` function to make sure that checker properly gets errors.

Reply:
@derrix060 Hi! Please, tell me, would you like to add extended test case with `async for`?